### DISCUSSION TITLE
Add DjangoGirls podcast to the website

### DISCUSSION THREAD

**Discussion by shante665** [DjangoGirls/djangogirls#835] [Ideas]
Can we add the Djangogirls podcast to the website. this could be part of the DjangoGirls story page.

https://anchor.fm/djangogirls
**RSS Feed**: https://anchor.fm/s/7aa83714/podcast/rss

Available on multiple podcast platforms
**Apple Pod**: https://podcasts.apple.com/us/podcast/django-girls-podcast/id1602881437
**Google Pod**: https://podcasts.google.com/feed/aHR0cHM6Ly9hbmNob3IuZm0vcy83YWE4MzcxNC9wb2RjYXN0L3Jzcw?ep=14
**Amazon Music**: https://music.amazon.ca/podcasts/afe571fc-bc45-4fcb-871f-3b5a4bb4560f/django-girls-podcast?ref=dm_sh_pXP60O3XBTy3Vb5o5JmSB1coQ
e.t.c

**Comment by ramonsaraiva** [comment]
Shame on me that I didn't know about this beautiful podcast! I love the idea of adding it to the website.
`/story/` would be a good place to reference it, and maybe even a small section/widget on the homepage!

Would love to know what the other contributors think 😻

**Comment by amakarudze** [comment]
I was thinking more on the home page. The `/story/` page is already filled with lots of pictures, am not sure where exactly we would put this on the page given it's current layout.

**Comment by ramonsaraiva** [comment]
Seems like there's enough vertical space under "Stay up to date" (or shifting it and putting it above) for a small "Latest from our podcast" section with a `anchor.fm/spotify` play button widget.. Wdyt?

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/5559120/211154603-ae326ddb-599d-4f99-9425-cb4e5202846e.png">

**Reply by amakarudze** [reply]
Yes, either would work @ramonsaraiva.
